packages/expo/src/components/Button.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { View, Alert, Button as SignInButton } from "react-native";
```

Comment:
The import name should not be specific.
We should use a generic name since we're going to use it all across the app.

```suggestion
import { View, Alert, Button as ButtonComponent } from "react-native";
```

Reply:
ok